### ISSUE TITLE
DRF support

### ISSUE THREAD

**Issue by justquick** [justquick/django-activity-stream#502]
django rest framework support for Follow/Action in actstream mod
maybe try token auth in runtests

**Comment by justquick** [comment]
yes ive reached out to @ahawker via email about collaborating but havent gotten a response. failing that i will take what he started an adapt it into the project. a first round prototype has been pretty promising but has some more things to work out

**Comment by auvipy** [comment]
just start with draft, we can improve it over time. https://github.com/Ian-Foote/rest-framework-generic-relations & https://www.django-rest-framework.org/api-guide/relations/#generic-relationships are helpful refs.
